.github/workflows/actions.yml

```diff
@@ -34,6 +34,14 @@ jobs:
           brew install swiftformat
           brew install xcbeautify
 
+      - name: Get cached Swift Packages managed by Xcode
+        uses: actions/cache@v2
```

Comment:
Any reason not to use `@v3` version?

Here is an example:
https://stackoverflow.com/a/72276931

Reply:
Not really, wasn't aware of v3, just used my usual script, thanks, bumping it now! 🙏🏻